### DISCUSSION TITLE
MEETING: efmnhtransactions Working Group - 11 January 2023

### DISCUSSION THREAD

**Discussion by rondlg** [fieldmuseum/EMu-Documentation#741] [MEETINGS]
# Agenda: Transactions working group
## CITES Update

- Mammals and Exhibitions request for this to show through. For this to work the data needs to be present in the taxonomy module. It is not yet.
- Mammals currently indicate this in the loan documentation currently.
- Insects (& IZ) has an indicator that shows up  in their summary string currently 
- Pulls from taxonomy / Red Data Tab

## Update from Anthro/Repatriation

- Can fields in Tasks be used to generate indicator icons? If they can, we'll use the Repatriation tree icon.
- A repatriated item will be deaccessioned in the same process as with other collections items.
- New transaction type aligned with “deaccession” for repatriation.
- TasDescription_tab can be used – but value in Task Description needs to exactly match (multiple rows with different values is ok) – e.g. currently if Task Description = “Repatriation”, record will display Repatriation tree icon:

## Multiple invoices versus closed loan?
- Questions:
> Can we track “work” per year with open and closed (and partial return dates) with a single invoice
> Can we track multiple permits (CITES, 3-177) for incoming and outgoing movements in transactions (or with emovements)
> Did we including linking transactions via transfer table
> Do we want to allow linking of Transactions?
- [efmntransactions structural changes document](https://docs.google.com/document/d/14lb2J9HDskjX_WYOYdLfMZuiYZYi7JRMCfdGX4OPbMk)

## Uncatalog object return tracking
- See ticket #57797
- Possible but not probable. Field is an open text field.
- Development request: Add taxonomy attachment field to uncatalogued items 

## Report  Request Mammals 
- adding geography to their invoice

## Homework

- [JJ] Synthesize a single invoice workflow addressing concerns raised in the questions above.
- Review efmnhtransactions structural changes to ensure appropriate development for this is noted
- Add Taxonomy and Geography links to uncatalogued objects tab.: DONE
- Adam (& ?) - get a GitHub account.

**Comment by smochel1716** [comment]
Fishes enters incoming loans from other institutions. Fishes doesn't make another loan for the return of that loan.  But we do have to change fields to print out a report that works for the return and then after printing the report, I switch them back.  Fields to change:  Transaction type - originally entered as incoming but changed to returning loan for return invoice.

**Comment by jmata8895** [comment]
Previously, Herps has either just closed the Incoming loan invoice or switched the transaction type from "incoming loan" to "returning loan" and then closed that invoice.

**Comment by ElanaDux** [comment]
Anthro uses just one transaction record per loan

**Reply by AnnaBalla** [reply]
Botany uses one number to create incoming loan and uses the same number when returning. If it's a full return we close it and enter a "closed" date

**Comment by cmcgarrity** [comment]
Test

**Comment by kjakymec** [comment]
Inverts enters incoming loans from other institutions. Inverts doesn't make another loan for the return of that loan. (As far as KGJ knows...)

**Comment by jwadleighfmnh** [comment]
**Incoming Loan EMu Workflow - Insects**
Single transaction record - Open (stays open until sent back)
Transaction Type: Incoming loan 
Other Institution # - The source institution's invoice or reference number.
Institution & Contact person - Source institution and source loaner
For Use By - Our staff that has borrowed the loan
Authorized By - Source Loaner
Objects (U) - Listed objects included in the loan

We send the incoming loan invoice to the source loaner and attach their invoices to multimedia.

When returning the loan we close the same transaction and send a return invoice.
